### PR TITLE
PDA-1359 Use PDA container with Kafka container

### PR DESCRIPTION
**Pull request by awang-parasoft**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    ports:
+      - "9093:9093"
+    volumes:
+      - "kafka_data:/bitnami"
+    environment:
+      - KAFKA_CFG_ZOOKEEPER_CONNECT=zookeeper:2181
+      - ALLOW_PLAINTEXT_LISTENER=yes
+      - KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=INTERNAL:PLAINTEXT,EXTERNAL:PLAINTEXT
+      - KAFKA_CFG_LISTENERS=INTERNAL://:9092,EXTERNAL://:9093
+      - KAFKA_CFG_ADVERTISED_LISTENERS=INTERNAL://kafka:9092,EXTERNAL://${EXTERNAL_HOSTNAME:-localhost}:9093
```
Comment (yshi-parasoft) [inline docker-compose.yml:33]: Should we also expose the port for configuring external server?

Comment (yshi-parasoft) [reply docker-compose.yml:33]: Also, can we connect to external Kafka server when only running a pda container? (like using application.properties). I think the docker-compose should be only used when we use Kafka in docker.

Comment (awang-parasoft) [reply docker-compose.yml:33]: > Should we also expose the port for configuring external server?

The `EXTERNAL` here is a Kafka listener in the Kafka server(Docker) used for connection from outside of this Docker network. It's not external Kafka server.

See: [Kafka Listeners - Explained](https://rmoff.net/2018/08/02/kafka-listeners-explained/)

Comment (awang-parasoft) [reply docker-compose.yml:33]: > Also, can we connect to external Kafka server when only running a pda container? (like using application.properties). I think the docker-compose should be only used when we use Kafka in docker.

Yes, there's no need to use Docker Compose if we don't need the Kafka in Docker. We can configure the Kafka server, and specify the Kafka server in PDA `application.properties` config file and start PDA container with a `docker run` command as usual.

**Pull request merged**